### PR TITLE
Successfully create cluster. 

### PR DESCRIPTION
**Pull request by liuruiyiyang**
Add CRD of Broker and MetaService which define the RocketMQ cluster to automatically deploy a RocketMQ cluster on Kubernetes cluster.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,132 @@
...
+apiVersion: cache.example.com/v1alpha1
+apiVersion: rocketmq.operator.com/v1alpha1
+kind: MetaService
+metadata:
+  name: meta-service
+spec:
+  # size is the the name service instance number of the name service cluster
+  size: 2
+  # metaServiceImage is the customized docker image repo of the RocketMQ name service
+  metaServiceImage: 2019liurui/rocketmq-namesrv:4.5.0-alpine
```
Comment (vongosling) [inline README.md]: 2019liurui seems not a reasonable name.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,132 @@
...
+  # size is the number of the broker cluster, each broker cluster contains a master broker and [slavePerGroup] slave brokers.
+  size: 2
+  # nameServers is the [ip:port] list of name service
+  nameServers: 192.168.130.33:9876;192.168.130.34:9876
+  # replicationMode is the broker slave sync mode, can be ASYNC or SYNC
+  replicationMode: ASYNC
+  # slavePerGroup is the number of each broker cluster
+  slavePerGroup: 2
+  # brokerImage is the customized docker image repo of the RocketMQ broker
+  brokerImage: 2019liurui/rocketmq-broker:4.5.0-alpine
```
Comment (vongosling) [inline README.md]: Refer to the above comment.

File: build/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,42 @@
+#FROM registry.access.redhat.com/ubi7-dev-preview/ubi-minimal:7.6
+FROM openjdk:8-alpine
```
Comment (vongosling) [inline build/Dockerfile]: No licence header

File: README.md (hunk 1 above)
Comment (liuruiyiyang) [reply README.md]: ```2019liurui``` has been replaced by ```rocketmqinc```.

File: README.md (hunk 2 above)
Comment (liuruiyiyang) [reply README.md]: fixed.

File: build/Dockerfile (hunk 3 above)
Comment (liuruiyiyang) [reply build/Dockerfile]: ASF header added.

**Pull request merged**